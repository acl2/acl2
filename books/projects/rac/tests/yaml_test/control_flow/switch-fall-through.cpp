// RAC begin

int foo(int a) {
  int b = 0;

  switch (a) {
  case 1:
    b = 1;
  case 2:
    b = 2;
    break;
  default:
    b = 3;
  }

  return b;
}

// RAC end
