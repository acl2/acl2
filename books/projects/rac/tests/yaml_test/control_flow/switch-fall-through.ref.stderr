switch-fall-through.cpp:7-8 (2-10): No break at the end of case
  7 |   case 1:
    |   ^^^^^^^^
  8 |     b = 1;
    | ^^^^^^^^^^
